geth/params/config_test.go

```diff
@@ -29,6 +29,7 @@ var loadConfigTestCases = []struct {
 			"WSPort": 8546,
 			"IPCEnabled": true,
 			"WSEnabled": false,
+			"HTTPEnabledMode": false,
```

Comment:
The new flag is now named `RPCEnabledMode`, not `HTTPEnabledMode` and this setting has no effect as the default value is false. I'd rename it to the appropriate name for consistency here, below in this file and in other configs.
Could you also write a test with `"RPCEnabledMode": true` to be confident this flag is now working?